.appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: 3.3.0.{build}
```

Comment:
Should we take this down a notch?

Reply:
It was set to match the [version in Directory.Build.props](https://github.com/WiseTechGlobal/BulkAnalysisRunner/blob/a07ad55e7ea55f9f5b94acd851ad61daa679b3c9/Directory.Build.props#L12) which was in turn set to match the Roslyn version.